docs/pages/resources/chapters.sk.mdx

```diff
@@ -0,0 +1,51 @@
+# Kapitoly (chapters)
```

Comment:
also here and on the other pages, do we really need the english translation of the resource keys?